Redo Web5 quickstart (#1422)
TBD54566975/developer.tbd.website: site/docs/web5/_quickstart-09-parse-vc.mdx

import CodeSnippet from '@site/src/components/CodeSnippet';

<section class="docs-section">

## 7. Parse your stored VC

> After creating and storing a Verifiable Credential (VC) as a JSON Web Token (JWT), users can convert it back to JSON format using the `parseJwt()` method. 

Reviewer: ```suggestion
Upon receiving a Verifiable Credential (VC) as a JSON Web Token (JWT), applications can convert it back to JSON format using the `VerifiableCredential.parseJwt()` method. 
```
